tests/test_notation_sets.py

```
@@ -0,0 +1,182 @@
+    "natural_transformation": {
+        "symbol": "α: F ⇒ G",
+        "meaning": "mapping between functors F and G"
+    },
+    "opposite_category": {
+        "symbol": "C^op",
+        "meaning": "category C with all arrows reversed"
+    },
+    "coproduct": {
+        "symbol": "A ⊔ B",
+        "alt_symbol": "A + B",
+        "dual_of": "A × B",
```

Ruff (0.16.2)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)